### DISCUSSION TITLE
[BUG]: API returning translations instead of the main core parameters of the listing

### DISCUSSION THREAD

**Discussion by yovasx2** [etsy/open-api#714] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

API returning translated (english) title, description and tags instead of the main core parameters of the listing shown in the shop manager (spanish)

### Steps to reproduce

I have a shop in spanish as default language (checked in shop manager) and API is returning both languages (es & en-US, this is weird)
then I created a published product with title, tags and description in Spanish:
<img width="1351" alt="Screen Shot 2022-09-21 at 20 58 38" src="https://user-images.githubusercontent.com/3578401/191642534-cf9e88c4-bff6-4c0c-a73d-9f5e0cfb0a17.png">
when I hit the get listing endpoint (https://developers.etsy.com/documentation/reference/#operation/getListing) instead of getting the Spanish attributes, I am getting the translations in English
 
<img width="638" alt="Screen Shot 2022-09-21 at 20 59 02" src="https://user-images.githubusercontent.com/3578401/191642739-97f8784a-322a-475c-bba1-21fd41e6eabe.png">
a 

### Expected behavior

Get listing endpoint must deliver the same info shown in the shop manager (spanish)

### Additional context

language of the shop is es and en-US, language in the product is en-US

Our customers have german and french core languages and english translations but they got their products automatically translated into English when hitting the api

**Comment by devtronic** [comment]
@etsyachristensen: Any news to this problem? Is there a workaround to receive the original values (german in our case).

**Comment by etsyachristensen** [comment]
@yovasx2 and @devtronic Can you provide a few listing ids where this is currently happening? I want to have our translation team look into them.

**Comment by devtronic** [comment]
Reported by one of our users:
- https://www.etsy.com/listing/921715474
- https://www.etsy.com/listing/251710635

**Comment by etsyachristensen** [comment]
@yovasx2 and @devtronic We are going to convert this to a discussion. We are still tracking this as an outstanding issue.

**Comment by djfp** [comment]
Hello and happy holidays! We ran into this issue just before Xmas in conjunction with Vela with apocalyptic data loss in our [German shop](https://www.etsy.com/de/shop/SeegangBerlin). 
All German originals (titles, descriptions and tags (if edited) have been replaced by their english translations, when the product was published from Vela back to Etsy. Almost 200 listings were affected!
 
i.e we assigned a group of products to a new section was sufficient to overwrite Title and Artist.

We are still in repair mode and data recovery mode over here - this bug is a major blocker and must be fixed asap.

**Comment by diegodleon** [comment]
This continues to happen as of today. Any news on a potential fix/workaround?

Seems like any time a non EN-US default language shop adds EN-US to its languages the translations in EN-US override the shops default language in your API.

**Reply by diegodleon** [reply]
To reply to myself the only feasible workaround so far is asking Etsy users to turn off EN-US from their languages if their shop is default to any other language.

**Comment by nateswart** [comment]
We are tracking what appears to be the same issue in the following discussion: https://github.com/etsy/open-api/discussions/868

If this is indeed the same issue, there is a workaround that you can force a translation value by passing the `langauge=de` query parameter to force the `de` translation.

**Comment by diegodleon** [comment]
This seems to be fixed now, fix came out in https://github.com/etsy/open-api/releases/tag/3.0.0-general-release-2023-03-16

Thanks!
